src/main/java/uk/gov/justice/digital/hmpps/whereabouts/repository/VideoLinkAppointmentRepository.kt

```diff
@@ -2,11 +2,12 @@ package uk.gov.justice.digital.hmpps.whereabouts.repository
 
 import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkAppointment
+import java.util.Optional
 
 interface VideoLinkAppointmentRepository : CrudRepository<VideoLinkAppointment, Long> {
   override fun findAll(): Set<VideoLinkAppointment>
 
   fun findVideoLinkAppointmentByAppointmentIdIn(appointmentIds: Set<Long>): Set<VideoLinkAppointment>
-  fun findOneByAppointmentId(appointmentId: Long): VideoLinkAppointment
+  fun findOneByAppointmentId(appointmentId: Long): Optional<VideoLinkAppointment>
```

Comment:
meh.. with kotlin tend to use nullable return rather than Optional `VideoLinkAppointment ?`